### ISSUE TITLE
Updating your README.md file and adding CODE_OF_CONDUCT.md

### ISSUE THREAD

**Issue by Sowham-3098** [IBM/carbon-ui-builder#158]
I want to add How to Contribute ,as it will be helpful for beginners and also adding contributers profiles into it and also want to add CODE_OF_CONDUCT.md

**Comment by zvonimirfras** [comment]
Hi Sowham (@Sowham-3098)!

We already have a [CONTRIBUTING.md](https://github.com/IBM/carbon-ui-builder/blob/main/CONTRIBUTING.md) and code of conduct should be inspired by [this](https://github.com/IBM/carbon-components-angular/blob/master/.github/CODE_OF_CONDUCT.md).

Is that what you had in mind?

**Comment by Sowham-3098** [comment]
Yeah! I also want to add all contributors profile images in main readme.md and it will automatically update when anyone contribute in this repo.
